tests/TalesRegistryTest.php

```diff
@@ -42,12 +42,11 @@ public function prepareTemplate( Text_Template $template ) {
     }
 
     /**
-     * @runInSeparateProcess
```

Comment:
I was unable to get this passing on a separate process, but it's ok in the main one now